plugins/woocommerce-blocks/assets/js/icons/library/customer-account-style-line.tsx

```diff
@@ -4,11 +4,24 @@
 import { SVG } from '@wordpress/primitives';
 
 const customerAccountStyleLine = (
-	<SVG xmlns="http://www.w3.org/2000/svg" viewBox="0 0 16 16">
+	<SVG
+		width="12"
+		height="13"
+		viewBox="0 0 12 13"
+		fill="none"
+		xmlns="http://www.w3.org/2000/svg"
+	>
+		<circle
+			cx="6"
+			cy="3"
+			r="2.25"
+			stroke="currentColor"
+			stroke-width="1.5"
+		/>
 		<path
```

Comment:
Same here - we should import and use the `Path` and `Circle` (and any other that apply) exports from the `SVG` package as well ([reference](https://github.com/WordPress/gutenberg/blob/trunk/packages/primitives/src/svg/index.js)).